tests/pass/import.ncl

```diff
@@ -1,3 +1,5 @@
 let Assert = fun l x => x || %blame% l in
 
-(import "imported.ncl" 3 == 3 | #Assert)
+let Imported = import "imported.ncl" in
+(Imported.f 3 == 3 | #Assert) &&
+(builtins.is_str Imported.foo | #Assert)
```

Comment:
This doesn't test typing at all, as there is not type annotation. What you want to do is:

- Revert this change. I think the original test was just testing the applicative syntax for imports, and it was fine. In general it's better to add tests that to alter previous one, because unless we 100% know what we are doing, we may change what they were actually testing without knowing it.
- Add a line with a let-binding for an import and type it. Basically just after line 169, add a variant:
  ```
  // Regression test for #430 (https://github.com/tweag/nickel/issues/430)
  let x = import "typed-import.ncl"
  in x : Num,
  ```